src/memory_provider.c

```diff
@@ -25,6 +25,22 @@ typedef struct umf_memory_provider_t {
     void *provider_priv;
 } umf_memory_provider_t;
 
+static int CTL_SUBTREE_HANDLER(by_handle_provider)(
+    void *ctx, enum ctl_query_source source, void *arg,
+    struct ctl_index_utlist *indexes, char *extra_name,
+    enum ctl_query_type query_type) {
+    (void)indexes, (void)source;
+    umf_memory_provider_handle_t hProvider = (umf_memory_provider_handle_t)ctx;
+    // umf_result_t (*ctl)(void *hProvider, int operationType, const char *name,
+    //                 void *arg, char *extra_name, enum ctl_query_type query_type);
+    hProvider->ops.ext.ctl(hProvider->provider_priv, /*unused*/ 0,
```

Comment:
As i said before extraname should be a name in this call, and extraname paprameters should be removed.

Reply:
Done